Sources/Apollo/ApolloClientProtocol.swift

```diff
@@ -79,3 +81,16 @@ public protocol ApolloClientProtocol: class {
                                                     queue: DispatchQueue,
                                                     resultHandler: @escaping GraphQLResultHandler<Subscription.Data>) -> Cancellable
 }
+
+// MARK: conveniences
+
+extension ApolloClientProtocol {
+  /// Clears all records from the cache store.
+  /// - Warning: The cache may be used by other clients. Calling this method will affect all clients using the same cache!
+  /// - Parameters:
+  ///   - callbackQueue: An optional queue to execute the completion handler on. The default is `.main`.
+  ///   - completion: An optional completion closure to execute when the cache has been cleared. The default is `nil`.
+  func clearCache(callbackQueue: DispatchQueue = .main, completion: ((Result<Void, Error>) -> Void)? = nil) {
```

Comment:
One thing to watch out for with this that's bitten me on other "convenience" implementations of protocols where i'm setting up default parameters is that if you add more parameters to `clearCache` in the future, this may wind up calling a different method than you're expecting. [Here's an example of where that bit me recently](https://github.com/apollographql/apollo-ios/pull/1450).

Reply:
I'd say default in the Apollo conformances - that's what most people are using rather than creating their own client.

Reply:
👍 Done!